include/mata/nfa/nfa.hh

```diff
@@ -262,6 +281,13 @@ public:
      */
     bool is_lang_empty(Run* cex = nullptr) const;
 
+    /**
+     * @brief Check if the language is empty using Tarjan's SCC discover algorithm.
```

Comment:
We should maybe say how `is_lang_empty()` works too.

Reply:
Ok, i wrote there something